### ISSUE TITLE
Train a simple neural network on MNIST

### ISSUE THREAD

**Issue by jdalch** [HAZAR00/PredictionConforme#2]
Train a simple neural network model on the MNIST dataset. Create a jupyter notebook with the necessary classes and functions. Add markdown cells in the jupyter notebook naming and explaining the different sections: data pre-processing, building the architecture of the model, training the model, plotting the results etc.

**Comment by josebadalmau** [comment]
- Add markdown cells explaining what each code cell is doing, and the choices you are making (hyper-parameters, architecture etc.)
- Visualize the results: plot the images along with their ground-truth and predicted label
- Create a functionality to load/save your model so that you don't have to train it every time
